design/rest-api-guideline.md

```diff
@@ -0,0 +1,434 @@
+# REST API Guideline
+
+* **Status**: Draft
+* **Author**: keycloak-team
```

Comment:
@pedroigor I'd suggest to add you as author (if you would like to) and add an extra field with "Co-authors" by putting people's name on it. I'd rather to know who to contact, I believe keycloak-team is too generic.

Reply:
@abstractj, I changed because this is teamwork even though I've started the document/discussions. In any case, we have our lists which people can use to contact us.

But if you want me to change the author, I can do it too.

Reply:
@pedroigor I'm not picky about it and it's up to you. But I'd rather to just follow the RFCs approach, where we know the authors.